frontend/packages/console-shared/src/components/dashboard/utilization-card/UtilizationItem.tsx

```diff
@@ -58,12 +59,16 @@ export const MultilineUtilizationItem: React.FC<MultilineUtilizationItemProps> =
     error,
     TopConsumerPopovers,
     byteDataType,
+    isOCSRendered = false,
+    chartType,
   }) => {
     const { t } = useTranslation();
     const { endDate, startDate, updateEndDate } = useUtilizationDuration();
-    const current = data.map((datum, index) =>
-      getCurrentData(humanizeValue, queries[index].desc, datum, dataUnits && dataUnits[index]),
-    );
+    const current = data.map(function(datum, index) {
+      return isOCSRendered && datum && datum.length > 1
+        ? String(datum[0].y + datum[1].y)
+        : getCurrentData(humanizeValue, queries[index].desc, datum, dataUnits && dataUnits[index]);
+    });
```

Comment:
```suggestion
    const current = data.map((datum, index) => getCurrentData(humanizeValue, queries[index].desc, datum, dataUnits && dataUnits[index]));
```
Change the query instead so it works with this.